src/elements/core/testing/private/visual-regression-snapshot.ts

```diff
@@ -20,61 +22,121 @@ function findElementCenter(snapshotElement: () => HTMLElement): [number, number]
   ];
 }
 
-export function testVisualDiff(snapshotElement: () => HTMLElement): void {
-  it('default', async function () {
-    await visualDiff(snapshotElement(), imageName(this.test!));
-  });
-}
+class VisualDiffSetup {
```

Comment:
Shouldn't we export the class?

Reply:
if you need to import the types like when you want to pre define the setup new VisualDiffSetup()? probably not needed...

Reply:
I think in that case we would need to reconsider the API again and if necessary just add the export keyword.
Or would you prefer to have it exported per default?

Reply:
nono, all ok :)